### DISCUSSION TITLE
Error in german POS-Tagging

### DISCUSSION THREAD

**Discussion by Ben-Bock** [explosion/spaCy#9006] [Help: Other Questions] [answered]
Hallo, vielen Dank für die super Arbeit!

Ich habe heute bei folgendem Satz: "**Dort gibt es sehr gutes Essen, nur die Sauberkeit war grottig!**" bemerkt, dass "sehr gutes Essen," und "nur die Sauberkeit" jeweils als ein einziges NOUN getagged werden. Das ist bei beiden deutschen Modellen der Fall. Ist leicht in dem gehosteten displacy nachzuvollziehen. 

Vielen Dank nochmals, 

Ben

**Comment by adrianeboyd** [comment, marked as answer]
Hi, the demo has a feature that merges noun chunks. If you uncheck "Merge Phrases" and run it again you can see the original word-level parse.

**Reply by Ben-Bock** [reply]
Cool, thank you for the explanation. Can it be that this is activated in the German model as default pipeline component? I have not extra activated that but get the same results in code. Thank you again!
